### PR TITLE
Collapse inputs and Dashboard confirmation

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] Automatically collapse inputs view if there are no inputs
- [x] Ask for confirmation before going back to dashboard (it might be pressed accidentally and closing some studies can take a while)

HideInputs

DashbaordConfirmation

### CODE REVIEW

**Review by mguidon — APPROVED** [review]
Very useful

**Comment by KZzizzle** [comment]
very cool. Someone whose initials are KZ was annoying and asked for the ability to disable the dashboard confirmation popup in settings.

**Review by KZzizzle — APPROVED** [review]

**Review by GitHK — COMMENTED** [review]
Could you make it so that if I press enter on the confirmation it will be like clicking "Yes"?

**Review by pcrespov — APPROVED** [review]

File: services/web/client/source/class/osparc/component/metadata/ServicesInStudy.js [hunk 1]
```diff
@@ -27,38 +27,69 @@ qx.Class.define("osparc.component.metadata.ServicesInStudy", {
   construct: function(studyData) {
     this.base(arguments);
 
-    this._setLayout(new qx.ui.layout.Grid(10));
+    const grid = new qx.ui.layout.Grid(10);
+    grid.setColumnFlex(this.self().gridPos.name, 1);
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/component/metadata/ServicesInStudy.js:31]: ``this.self``?? sounds like two languages together to identify current instance.  you cannot just ``this.gridPos``?

Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/metadata/ServicesInStudy.js:31]: That's the static methods/variables calling way. I think that what you suggest only works for static classes.

**Review by sanderegg — APPROVED** [review]
great!

**Pull request merged**